docs/layouts.rst

```diff
@@ -167,7 +169,7 @@ If you want to extend multiple parents, it looks like this::
     extends:
       - socrates
       - plato
-    x: += 50    # evaluates to 150
+    x: += 50    # evaluates to 250 from plato
```

Comment:
I think this is correct, and the example a bit further down shows the opposite case.